website/docs/r/is_security_group_target.html.markdown

```diff
@@ -9,10 +9,7 @@ description: |-
 
 # ibm_is_security_group_target
```

Comment:
when checked in https://registry.terraform.io/tools/doc-preview tool,

Note

"IBM Cloud terraform provider currently provides both a standalone" portion has not come right, Can you please modify this

Reply:
This is aligned to the above paragraph, do we really need this ?

Reply:
Please check whether we need it or not, and remove if not required.
if required we need to add like   ~>**Note:**  section

Reply:
I think Note section is aligned as per other resources are using top section Note like BaaS.

Reply:
There is some alignment issues with provider section - please fix it.

Reply:
done.